projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -3433,117 +3433,6 @@ public string MotherEthnicityLiteral
             }
         }
 
-        /// <summary>Mother's Race values.</summary>
-        /// <value>the mother's race. A tuple, where the first value of the tuple is the display value, and the second is
-        /// the IJE code Y or N.</value>
-        /// <example>
-        /// <para>// Setter:</para>
-        /// <para>ExampleBirthRecord.MotherRace = {NvssRace.BlackOrAfricanAmerican, "Y"};</para>
-        /// <para>// Getter:</para>
-        /// <para>string boaa = ExampleBirthRecord.RaceBlackOfAfricanAmerican;</para>
-        /// </example>
-        [Property("MotherRace", Property.Types.TupleArr, "Race and Ethnicity Profiles", "Mother's Race", true, VR.IGURL.InputRaceAndEthnicity, true, 38)]
-        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='inputraceandethnicityMother')", "")]
-        [FHIRSubject(FHIRSubject.Subject.Newborn)]
-        public Tuple<string, string>[] MotherRace
```

Comment:
I think MotherRace should stay in this file. See other comments for alternative approach where we can keep the MotherRace fields within the BFDR library